DiscordChatExporter.Core/Discord/Data/Thread.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Linq;
+using System.Text.Json;
+using DiscordChatExporter.Core.Discord.Data.Common;
+using DiscordChatExporter.Core.Utils.Extensions;
+using JsonExtensions.Reading;
+
+namespace DiscordChatExporter.Core.Discord.Data;
+
+// https://discord.com/developers/docs/resources/channel#channel-object-example-thread-channel
+public partial record Thread(
```

Comment:
Probably should call it `ThreadChannel` to avoid naming ambiguity with `System.Threading.Thread`. That way you don't have to use `Data.Thread` everywhere.

Reply:
Resolved in 92897399938aac018696b429597828cde5873238.